frontend/lib/api/polling_station/PollingStationListProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import { PollingStation, usePollingStationListRequest } from "@kiesraad/api";
+
+export interface iPollingStationListProviderContext {
+  pollingStations: PollingStation[];
+}
+
+export const PollingStationListProviderContext = React.createContext<iPollingStationListProviderContext | undefined>(
+  undefined,
+);
+
+export interface PollingStationListProviderProps {
+  children: React.ReactNode;
+  electionId: number;
+}
+
+export function PollingStationListProvider({ children, electionId }: PollingStationListProviderProps) {
+  const { data, loading } = usePollingStationListRequest({ election_id: electionId });
+
+  if (loading) {
+    return null;
+  }
+
+  if (!data) {
+    return <div>Error no polling stations data</div>;
+  }
```

Comment:
Maybe it's good to make this a little more friendly and in Dutch. Something like: `<div>Er zijn nog geen stembureaus aangemaakt voor deze verkiezing.</div>`

Reply:
Current solution is consistent with `ElectionListProvider`. If we want to make the error more friendly, we should also have more on the page than just the error message. And I think that's in scope of #277 Improve error pages design.